demo/index.md

```diff
@@ -33,18 +33,34 @@ The `auro-sidenavsection` is an extension of the `auro-accordion` can be used to
 The `auro-sidenavitem` is an extension of the `auro-hyperlink` and used to create the navigational links in the `auro-sidenav`. You can use the expected `href` and `target` attributes to properly enable and style the links.
 
 Please refer to the below example to see how to use these components.
-<!-- AURO-GENERATED-CONTENT:END -->
-
-## Examples
-
-#### Default
+<!-- AURO-GENERATED-CONTENT:END -->
 
+## Examples
+
+<h4 id="default">Default</h4>
 The following example illustrates a common use case for the `sidenav` element that includes a navigation header, an accordion sub-nav action, as well as a link that will open to an external URL.
 
 <div class="exampleWrapper">
   <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/full.html) -->
   <!-- The below content is automatically added from ./../../apiExamples/full.html -->
-  <auro-sidenav>
+  <script>
+    // This design token variable can be whatever/from wherever you want
+    const DESIGN_TOKEN_VARIABLE = '--ds-grid-breakpoint-sm';
+    const smallBreakpointPx = getComputedStyle(document.documentElement).getPropertyValue(DESIGN_TOKEN_VARIABLE)
+    const mediaQueryString = `(min-width: ${smallBreakpointPx})`
+    const windowMediaMatcher = window.matchMedia(mediaQueryString);
+    const toggleSidenavCollapse = (e) => {
+      const auroSidenavExample = document.getElementById("collapsible-example")
+      if (!e.matches) {
+        auroSidenavExample.setAttribute('collapsible', "")
+      } else {
+        auroSidenavExample.removeAttribute("collapsible")
+      }
+    };
+    windowMediaMatcher.onchange = toggleSidenavCollapse;
+    window.onload = toggleSidenavCollapse;
+  </script>
```

Comment:
I can't put my finger on it, but there is something wrong with this example. When running this in the browser, with desktop (full width) and a refresh, the sidenav is collapsed. 

What I see, regardless of screen width, the sidenav is expanded and the `collapsible` attribute is added. As I resize the width I see the `collapsible` attribute added and removed. 

As previously discussed, this element should really be mobile-first. E.g. the `collapsible` state should be collapsed by default. Then the proposed new name of `isToggle` and the accordion feature is REMOVED as the screen size grows. 

So the default HTML template would have auro-accordion and the slot-change would REMOVE the accordion. 

For the default demo, I would reduce the scope of the elements within the nav, maybe just `<auro-sidenavitem href="/content/travel-info/pets">Pet travel overview</auro-sidenavitem>`

The remaining demos will address the rest of the core features.

Reply:
Please also update `#### selected` to be sentence case.

Reply:
Quick follow up on the `isToggle` naming - seems like that's a little counter-intuitive (we had a tough time wrapping our heads around the functionality) as we're doing the "wrapped in accordion by default" functionality now with mobile first consideration.

we're counter-suggesting `isStatic` since that implies it's not going to toggle anymore and renders as-is no matter what.

Workflow is -> `auro-sidenav` is 100% accordion all the time on mobile and desktop until you give it the `isStatic` attribute

Reply:
Quick follow up on the `isToggle` naming - seems like that's a little counter-intuitive (we had a tough time wrapping our heads around the functionality) as we're doing the "wrapped in accordion by default" functionality now with mobile first consideration.

we're counter-suggesting `isStatic` since that implies it's not going to toggle anymore and renders as-is no matter what.

Workflow is -> `auro-sidenav` is 100% accordion all the time on mobile and desktop until you give it the `isStatic` attribute